__tests__/integration/multisig.test.js

```diff
@@ -122,57 +123,14 @@ describe('send tx (HTR)', () => {
     expect(sig1).toBeTruthy();
     const sig2 = await wallet2.getSignatures(txHex);
     expect(sig2).toBeTruthy();
+    loggers.test.insertLineToLog('multisig[should fail lt minsig]: signatures', { signatures: [sig1, sig2] });
 
     // try to send
     response = await TestUtils.request
       .post('/wallet/tx-proposal/sign-and-push')
       .send({ txHex, signatures: [sig1, sig2] })
       .set({ 'x-wallet-id': wallet1.walletId });
-    console.log(JSON.stringify(response.body));
-
-    expect(response.status).toBe(200);
-    expect(response.body.success).toBe(false);
-  });
-
-  it('Should fail to send a transaction with more than max signatures', async () => {
```

Comment:
This test was removed because the sign and push api now removes any excess signatures so this behavior would not fail anymore.

Reply:
This was changed, so should we add this test back?

Reply:
Actually I've seen that you already have a test with more than minSignatures, so should be okay to remove this test, right?